draft-ietf-teep-protocol.md

```diff
@@ -1110,7 +1110,7 @@ update = [
   type: TEEP-TYPE-update,
   options: {
     ? token => uint,
-    ? tc-list => [ + SUIT_Component_Identifier ],
```

Comment:
this should be SUIT_Component_Identifier, to match the type of unneeded-tc-list in the QueryResponse message.  It would be ok to rename tc-list here (e.g., to unneeded-tc-list or delete-tc-list).